libs/blocks/merch-card/merch-card.js

```diff
@@ -0,0 +1,218 @@
+/* eslint-disable prefer-destructuring */
+import { decorateButtons } from '../../utils/decorate.js';
+import { loadStyle, getConfig, createTag } from '../../utils/utils.js';
+import { addBackgroundImg, addWrapper, addFooter } from '../card/cardUtils.js';
+import { decorateLinkAnalytics } from '../../martech/attributes.js';
+
+const {
+  miloLibs,
+  codeRoot,
+} = getConfig();
+const base = miloLibs || codeRoot;
+const SECURE_TRANSACTION_IMG = `<img class="secure-transaction-icon-img" loading="lazy" alt="secure-transaction-icon icon" src="${base}/blocks/merch-card/img/secure-transaction-icon.svg" height="12" width="15">`;
+
+const SEGMENT_BLADE = 'SegmentBlade';
+const SPECIAL_OFFERS = 'SpecialOffers';
+const PLANS_CARD = 'PlansCard';
+
+const getPodType = (styles) => {
+  const cardTypes = {
+    segment: SEGMENT_BLADE,
+    'special-offers': SPECIAL_OFFERS,
+    plans: PLANS_CARD,
+  };
+  const authoredType = styles?.find((style) => style in cardTypes);
+  return cardTypes[authoredType] || SEGMENT_BLADE;
+};
+
+const createDescription = (rows, cardType) => {
+  const descriptions = rows.slice(0, rows.length - 1);
+  const descriptionWrapper = createTag('div');
+  descriptionWrapper.classList.add(`consonant-${cardType}-description`);
+  descriptions?.forEach((description) => {
+    descriptionWrapper.appendChild(description);
+  });
+  return descriptionWrapper;
+};
+
+const createTitle = (titles, cardType) => {
+  const titleWrapper = createTag('div');
+  titleWrapper.classList.add(`consonant-${cardType}-title`);
+  titles?.forEach((title) => {
+    titleWrapper.appendChild(title);
```

Comment:
no need for `{`, `}`, it's just one line

Reply:
done